app/views/datagrid/_head.html.erb

```diff
@@ -1,8 +1,30 @@
 <tr>
   <% grid.html_columns(*options[:columns]).each do |column| %>
-    <th class="<%= datagrid_column_classes(grid, column) %>">
+    <%= tag.th(
+      class: {
+        # Adding HTML classes based on condition
+        # Consider maintaining consistency with datagrid/rows partial
+        "datagrid-order-active-asc": grid.ordered_by?(column, false),
+        "datagrid-order-active-desc": grid.ordered_by?(column, true),
+        column.html_class => column.html_class.present?,
+      },
+      "data-column": column.name
```

Comment:
Wdyt about also supporting passing `data` attributes to `column` definition?
Then using that with some method e.g. `column.html_data`.

From your Readme:
> It is more semantic and collision proof to use `data-*` attributes instead of classes for meta information from backend.

So since you’re now encouraging to use data attributes instead of `class`, IMO it’s logical to allow passing `data` attribute to column just like `class:` can be passed, and aligns with your philosophy of favoring data-* attributes over classes for metadata. 

I have a few grids where I pass `class:` to column definition to later use that in JavaScript. I would gladly use HTML data attributes for this.

Reply:
Good idea.
We may supporting everything with `tag_options` option (similar to `input_options` and `label_options` for `filter`):

``` ruby
column(:all, tag_options: {class: 'no-sort', "data-no-sort": true,  ...}, ...)
```

I was planning this for later release as this one is already big, but we can include it here if you it facilitates migration.

Reply:
I like `tag_options`. Let’s do a separate release later.